Sources/DistributedActors/Cluster/SWIM/Protobuf/SWIM+Serialization.swift

```diff
@@ -165,18 +165,18 @@ extension SWIM.Member: _ProtobufRepresentable {
 
     public func toProto(context: Serialization.Context) throws -> _ProtoSWIMMember {
         var proto = _ProtoSWIMMember()
-        guard let actorPeer = self.peer as? SWIM.Ref else {
-            throw SerializationError.unableToSerialize(hint: "Expected peer to be \(SWIM.Ref.self) but was \(self.peer)!")
+        guard let actorPeer = self.peer as? SWIM.Actor else {
+            throw SerializationError.unableToSerialize(hint: "Expected peer to be \(SWIM.Actor.self) but was \(self.peer)!")
         }
-        proto.id = try actorPeer.toProto(context: context)
+        proto.id = try actorPeer.id.toProto(context: context)
         proto.status = try self.status.toProto(context: context)
         proto.protocolPeriod = self.protocolPeriod
         return proto
     }
 
     public init(fromProto proto: _ProtoSWIMMember, context: Serialization.Context) throws {
         let id = try ActorID(fromProto: proto.id, context: context)
-        let peer = context._resolveActorRef(SWIM.Message.self, identifiedBy: id)
+        let peer = try SWIM.Actor.resolve(id: id, using: context.system)
```

Comment:
Yes, this is correct `(SomeActorType).resolve` is the way to do these now.

Nitpick on the name `Actor` as type name is a bit confusing I guess since there is an `protocol Actor` in stdlib

Reply:
It looks better than `SWIM.Actor` I think, it is a bit "meh" since a lot of types end up being called "Actor" if we go down that path... We'll find some better name -- maybe just SWIMActor or SWIMActorShell eventually?>?

In this PR just wanted to avoid the Actor` typealias